### PR TITLE
create a workflow for publishing the charm to Charmhub

### PR DESCRIPTION
**Pull request by matheushent**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+jobs:
+  ci-tests:
+    uses: ./.github/workflows/test.yaml
+
+  release-to-charmhub:
+    name: Release to CharmHub
+    needs:
+      - ci-tests
+    runs-on: ubuntu-24.04
```
Comment (jaimesouza) [inline .github/workflows/release.yaml:39]: Have you been able to test this on Ubuntu 24.04 yet? Maybe we should still consider 22.04 for a while.

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Release to Charmhub
+
+on:
+  workflow_dispatch:
```
Comment (fschuch) [inline .github/workflows/release.yaml:18]: I assume it is supposed to run from tags, so we select the version tag and the channeled?

File: .github/workflows/release.yaml (hunk 1 above)
Comment (matheushent) [reply .github/workflows/release.yaml:39]: you're right. I'm gonna change it

File: .github/workflows/release.yaml (hunk 2 above)
Comment (matheushent) [reply .github/workflows/release.yaml:18]: you select the tag from the GitHub UI itself. The channel is inputed in the UI as well

File: .github/workflows/release.yaml (hunk 1 above)
Comment (matheushent) [reply .github/workflows/release.yaml]: done [522734f](https://github.com/omnivector-solutions/charm-jobbergate-agent/pull/17/commits/522734f1db7ab8264a3d0cdf3bd3fe1c4ce3b0f6)

**Review by jaimesouza — APPROVED** [review]
Looks good to me!

**Review by fschuch — APPROVED** [review]

**Review by jaimesouza — APPROVED** [review]
+1

**Review by fschuch — APPROVED** [review]

**Pull request merged**